apps/storybook/src/Annotation.stories.tsx

```diff
@@ -1,14 +1,16 @@
-import { Annotation, DefaultInteractions, VisCanvas } from '@h5web/lib';
+import type { CanvasEvent } from '@h5web/lib';
+import { Annotation, useCanvasEvents } from '@h5web/lib';
+import { useRafState } from '@react-hookz/web';
 import type { Meta, Story } from '@storybook/react';
+import { useCallback } from 'react';
+import type { Vector3 } from 'three';
 
+import DefaultCanvas from './decorators/DefaultCanvas';
 import FillHeight from './decorators/FillHeight';
+import { formatCoord } from './utils';
 
 export const Default: Story = () => (
-  <VisCanvas
-    abscissaConfig={{ visDomain: [0, 41], showGrid: true }}
-    ordinateConfig={{ visDomain: [0, 20], showGrid: true }}
-  >
-    <DefaultInteractions />
```

Comment:
Moving this to a decorator so that it doesn't get re-rendered when the `FollowPointer` story re-renders. This is required to test `useCameraState` properly, because when the whole canvas re-renders, it also invokes all the `useFrame`s, so it becomes impossible to isolate updates made via `useMemo`.